scripts/testing/run_e2e_sap_btp_manager_secret_test.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+# The script requires BTP Manager to be installed and running in the cluster.
+# Run install_module.sh script before running this script.
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+echo -e "\n--- SAP BTP Manager secret customization test ---"
+
+# Set environment variables
+## Resources names
+NAMESPACE=kyma-system
+BTP_MANAGER_SECRET_NAME=sap-btp-manager
+SAP_BTP_OPERATOR_SECRET_NAME=sap-btp-service-operator
+SAP_BTP_OPERATOR_CLUSTER_ID_SECRET_NAME=sap-btp-operator-clusterid
+SAP_BTP_OPERATOR_CONFIGMAP_NAME=sap-btp-operator-config
+
+## Fake values for the secret
+CLIENT_ID=$(echo -n "client-id" | base64)
+CLIENT_SECRET=$(echo -n "client-secret" | base64)
+SM_URL=$(echo -n "sm-url" | base64)
+TOKEN_URL=$(echo -n "token-url" | base64)
+CLUSTER_ID="cluster-id"
+MANAGEMENT_NAMESPACE="management-namespace"
+ENCODED_CLUSTER_ID=$(echo -n ${CLUSTER_ID} | base64)
+ENCODED_MANAGEMENT_NAMESPACE=$(echo -n ${MANAGEMENT_NAMESPACE} | base64)
+
+## Save current resourceVersion of the resources to be updated
+SAP_BTP_OPERATOR_SECRET_RESOURCE_VERSION=$(kubectl get secret -n ${NAMESPACE} ${SAP_BTP_OPERATOR_SECRET_NAME} -o jsonpath="{.metadata.resourceVersion}")
+SAP_BTP_OPERATOR_CONFIGMAP_RESOURCE_VERSION=$(kubectl get configmap -n ${NAMESPACE} ${SAP_BTP_OPERATOR_CONFIGMAP_NAME} -o jsonpath="{.metadata.resourceVersion}")
+
+## Save current ID of the resource to be recreated
+SAP_BTP_OPERATOR_CLUSTER_ID_SECRET_ID=$(kubectl get secret -n ${NAMESPACE} ${SAP_BTP_OPERATOR_CLUSTER_ID_SECRET_NAME} -o jsonpath="{.metadata.uid}")
```

Comment:
The secret that should be eventually re-created with a new `cluster_id` should be located in the `$MANAGEMENT_NAMESPACE` namespace.

Reply:
AFAIK the secret `sap-btp-operator-clusterid` is created in release namespace.

Reply:
Correct, `sap-btp-operator-clusterid` secret is created in the release namespace: https://github.com/SAP/sap-btp-service-operator/blob/cca53330710b7c662001ac10e35f91016c27887d/main.go#L219